Ash.FrontEnd/src/global/style.less

```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;

```

Comment:
New design will have these sections span the whole width of their containers, so it doesn't need left and right margins.